### ISSUE TITLE
Новость все еще отображается для неактивной компании

### ISSUE THREAD

**Issue by vbezruchkin** [Fulcanellii/Intelliants#2]
1. Создал компанию, сделал неактивной
2. Создал новость, привязал к неактивной компании. Сама новость активна.

Условием было отображать только активные новости для активных компаний.

**Comment by akulmambetov** [comment]
То есть если сама новость активная, но нету активных компаний то ее не отображать? 12.08.2020, 20:44, "Vasily Bezruchkin" <notifications@github.com>:

Создал компанию, сделал неактивной
Создал новость, привязал к неактивной компании. Сама новость активна.

Условием было отображать только активные новости для активных компаний.

**Comment by vbezruchkin** [comment]
Да, новость отображается только когда новость активна и любая из привязанных компаний активна.

**Comment by akulmambetov** [comment]
Сейчас исправлю12.08.2020, 22:31, "Vasily Bezruchkin" <notifications@github.com>:
Да, новость отображается только когда новость активна и любая из привязанных компаний активна.

**Comment by akulmambetov** [comment]
Исправил12.08.2020, 22:31, "Vasily Bezruchkin" <notifications@github.com>:
Да, новость отображается только когда новость активна и любая из привязанных компаний активна.
